### PR TITLE
Add POJO-TESTER dependency and test Label and User classes

### PR DESCRIPTION
**Pull request by ThePnx**
Easy tool for bumping coverage 🌮 (and not only) http://www.pojo.pl/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 buildscript {
     repositories {
+        jcenter()
```
Comment (mateuszmarek) [inline build.gradle:3]: good one, i've read good things about jcenter vs mavenCentral just yesterday ! :)

**Review by mateuszmarek — APPROVED** [review]

File: src/test/java/rotateit/data/model/UserTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+public class UserTest {
+
+    @Test
+    public void testGettersSettersAndToString() {
+        assertPojoMethodsFor(User.class)
+            .testing(GETTER)
+            .testing(SETTER)
+            .testing(TO_STRING)
+            .areWellImplemented();
```
Comment (Vortim) [inline src/test/java/rotateit/data/model/UserTest.java:16]: awesome! :)

**Review by Vortim — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (ThePnx) [reply build.gradle:3]: :+1: for jcenter but I read about people having problems with missing libs in jcentral which ones were present in maven central, so I think we can have both of them.

File: src/test/java/rotateit/data/model/UserTest.java (hunk 2 above)
Comment (ThePnx) [reply src/test/java/rotateit/data/model/UserTest.java:16]: I like it to :)

**Pull request merged**